ui/components/app/modals/transaction-already-confirmed/transaction-already-confirmed.container.js

```diff
@@ -0,0 +1,36 @@
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { getBlockExplorerLink } from '@metamask/etherscan-link';
+import withModalProps from '../../../../helpers/higher-order-components/with-modal-props';
+import { hideModal } from '../../../../store/actions';
+import { getRpcPrefsForCurrentProvider } from '../../../../selectors';
+
+import TransactionAlreadyConfirmed from './transaction-already-confirmed.component';
+
+function mapDispatchToProps(dispatch) {
+  return {
+    hideModal: () => dispatch(hideModal()),
```

Comment:
`hideModal` is not needed here. It is [already defined by `withModalProps`](https://github.com/MetaMask/metamask-extension/blob/f8b59c75bd0c45db980fbedcbf39bfde75933a5b/ui/helpers/higher-order-components/with-modal-props/with-modal-props.js#L15).

Reply:
Good spot, removed in https://github.com/MetaMask/metamask-extension/pull/22943/commits/c23a8a0d973021d6500fad10e627446e7448ab9d